### DISCUSSION TITLE
Can i use vercel for commercial usage ?

### DISCUSSION THREAD

**Discussion by MrinmoyHaloi** [vercel/vercel#6590] [General] [answered]
Can I use vercel for commercial usage like hosting clients websites. Without charging for hosting from the client. (No reselling).

**Comment by fu-sen** [comment]
No. Free plan Hobby accounts are restricted to non-commercial personal use.
https://vercel.com/docs/platform/fair-use-policy#commercial-usage

**Reply by MrinmoyHaloi** [reply]
ok thanks

**Comment by leerob** [comment, marked as answer]
Vercel's free tier (e.g. your personal, always free account) is not available for usage on [paid, commercial projects](https://vercel.com/docs/platform/fair-use-policy#commercial-usage).

To clarify, you _can_ use Vercel for client's websites on a Pro plan. If you have clients, you should also be thinking about the "handoff" process when your contract/work is completed, in which you can transfer the Vercel Team to the client. We have lots of agencies and freelancers who use a Vercel team to create projects for their clients 😄
